docs/tutorials/tutorial-hdfs-logs-distributed-search-aws-s3.md

```diff
@@ -27,144 +27,201 @@ Before using Quickwit with an object storage, check out our [advice](../administ
+This step can also be done on your local machine. The `ingest` subcommand generates locally [splits](../overview/architecture.md) of 10 million documents and will upload them on your bucket. Concretely, each split is a bundle of index files and metadata files.
 
 :::
 
 
-You can check it's working by using `search` command and look for `ERROR` in `serverity_text` field:
+You can check it's working by using `search` subcommand and look for `ERROR` in `serverity_text` field:
 ```bash
-quickwit search --index-uri s3://path-to-your-bucket/hdfs_logs --query "severity_text:ERROR"
+./quickwit index search --index hdfslogs --config ./config.yaml --query "severity_text:ERROR"
 ```
 
+Now that we have indexed the logs and can search from one instance, It's time to configure and start a search cluster.
+
+## Start the cluster
+
+Quickwit needs a port `rest_listen_port` for serving the HTTP rest API via TCP as well as maintaining the cluster formation via UDP. Also, it needs `{rest_listen_port} + 1` for gRPC communication between instances.
 
-## Start your servers
+In AWS, you can create a security group to group these inbound rules. Check out the [network section](configure-aws-env.md) of our AWS setup guide.
+
+To make things easier, let's create a security group that opens the TCP/UDP port range [7100-7400]. Next, create three EC2 instances using the previously created security group. Take note of each instance's public IP address. 
 
-The command `serve` starts an http server which provides a [REST API](../reference/search-api.md).
-Run it on each of your instances.
+Now ssh into the first EC2 instance, install Quickwit, and [configure the environment](configure-aws-env.md) to let Quickwit access the index S3 buckets.
 
 ```bash
-quickwit serve --index-uri s3://path-to-your-bucket/hdfs_logs --peer-seed=ip1,ip2,ip3
+curl -L https://install.quickwit.io | sh
+cd quickwit-v*/
 ```
 
-You will see in your terminal the confirmation that the instance has joined the cluster. Example of such a log:
+Let's create the node configuration file by first running this snippet. Make sure you fill in the correct values before running in the EC2 terminal session. 
 
+```bash
+export S3_PATH=s3://{path/to/bucket}/indexes
+export IP_NODE_1={first-ec2-instance-public-ip}
+export IP_NODE_2={second-ec2-instance-public-ip}
+export IP_NODE_3={third-ec2-instance-public-ip}
 ```
-INFO quickwit_cluster::cluster: Joined. host_key=019e4c0e-165a-430d-8ef6-7b7a035decac remote_host=Some(172.31.66.143:8081)
+
+```bash
+# configuration for our first node
+echo "version: 0
+node_id: searcher-1
+metastore_uri: ${S3_PATH}
+default_index_root_uri: ${S3_PATH}
+searcher:
+  rest_listen_address: 0.0.0.0
+  rest_listen_port: 7100
+  peer_seeds:
+    - ${IP_NODE_2}:7200 # searcher-2
```

Comment:
I would avoid changing the ports. It's clearer to have all instances on the same ports.

Reply:
by the way, using the default port would be nice, so that we don't need to add the port in the peer_seeds.